documentation/staging/content/userguide/managing-fmw-domains/_index.md

```diff
@@ -62,7 +62,7 @@ following limitations currently exist for FMW Infrastructure domains:
 #### Obtaining the FMW Infrastructure image
 
 The WebLogic Kubernetes Operator requires patch 29135930.
-The standard pre-built FMW Infrastructure image, `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3`, already has this patch applied. For detailed instructions on how to log in to the Oracle Container Registry and accept license agreement, see this [document]({{< relref "/userguide/base-images/_index.md#obtain-images-from-the-oracle-container-registry" >}}).
+The standard pre-built FMW Infrastructure image, `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3`, already has this patch applied. For detailed instructions on how to log in to the Oracle Container Registry and accept license agreement, see this [document]({{< relref "/userguide/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}}).
```

Comment:
It looks like we need to search all locations of not only ":12.2.1.4" but also ":12.2.1.3", and work in the GA warning...

Reply:
Not done, I'll confer with Monica and Derek.